### ISSUE TITLE
Rancher container with wrong MTU

### ISSUE THREAD

**Issue by arbermarleku** [rancher/rancher#19119]
**What kind of request is this (question/bug/enhancement/feature request): BUG**

**Steps to reproduce (least amount of steps as possible):*Install a normal rancher. Mine is : v2.2.0-rc9*

**Result:*Rancher will start but with errors. Cause MTU is by default 1500*
![image](https://user-images.githubusercontent.com/41958760/54859020-dfb88600-4d08-11e9-9d65-2d951f7abdfc.png)

i skipped the rest of the questions and down below is the fix as workaround

back up first the docker.service located into /lib/systemd/system/docker.service

edit the following service do a nano /lib/systemd/system/docker.service

go and add this flag 👍

![image](https://user-images.githubusercontent.com/41958760/54859053-3920b500-4d09-11e9-91cb-701712151739.png)

restart the deamon and the service

run :
systemctl daemon-reload
service docker restart

have a nice day :)

**Comment by erikwilson** [comment]
Some more info on the issue, this is not related to any specific version of Rancher but a general system level networking problem which manifests as a catalog error. The specific MTU value is network dependent so there is no safe or universal value.

Since this is not Rancher specific I am labeling this as a feature request, where we should either provide documentation, a warning, or possibly ensuring the MTU is correct if it is an issue for RancherOS. One possibility is that in Rancher server or RKE we could do a ping test using the interface MTU size to ensure no packet loss, and emit a warning if there is.

**Comment by ak2766** [comment]
> edit the following service do a nano /lib/systemd/system/docker.service

No, no, no.  Do not edit system files.  Your changes will be tossed by the next doscker update.

Instead, do the following:

    $ sudo systemctl edit docker
Add the following text into the new file:

    -------------CUT BELOW THIS LINE-------------
    [Service]
    ExecStart=
    ExectStart=<copy from vendor service file> --mtu=1400
    -------------CUT ABOVE THIS LINE-------------
    $ sudo systemctl restart docker

Note: You have to zero out the ExecStart from the vendor provided service file before you can apply your own modififcations.

If you need to revert these changes to those provided by vendor, then:

    $ sudo systemctl revert docker
    $ sudo systemctl restart docker

If the version of systemd doesn't have the revert option:

    $ sudo rm -r /etc/systemd/system/docker.service.d
    $ sudo systemctl daemon-reload
    $ sudo systemctl restart docker
